extensions/community/CrazyGamesAdApi.json

```diff
@@ -0,0 +1,393 @@
+          "type": "string"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Check if CrazyGame SDK is loaded.",
+      "fullName": "Check if CrazyGame SDK is loaded",
+      "functionType": "Condition",
+      "name": "IsSDKLoaded",
+      "sentence": "Is CrazyGame SDK loaded?",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "eventsFunctionContext.returnValue = (CrazyGames != undefined);",
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Display CrazyGames Happy Time.",
+      "fullName": "Display CrazyGames Happy Time",
+      "functionType": "Action",
+      "name": "DisplayHappyTime",
+      "sentence": "Display CrazyGames Happy Time",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "try {",
+            "    const crazysdk = window.CrazyGames.CrazySDK.getInstance();",
+            "    crazysdk.init();",
+            "",
+            "    // Player beats a boss, reaches a high score, etc.",
+            "    crazysdk.happytime();",
+            "} catch(e) {",
+            "    console.log({e});",
+            "}"
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Check if user is using adblock.",
+      "fullName": "Check if user is using adblock",
+      "functionType": "Action",
+      "name": "ScanForAdBlockers",
+      "sentence": "Check if user is using adblock",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "try {",
+            "    function adblockDetection(event) {",
+            "        if (event.hasAdblock) {",
+            "            runtimeScene.getVariables().get(\"__CrazyGamesExtension\").getChild(\"adBlocker\").setBoolean(true);",
+            "        } else {",
+            "            runtimeScene.getVariables().get(\"__CrazyGamesExtension\").getChild(\"adBlocker\").setBoolean(false);",
+            "        }",
+            "    }",
+            "",
+            "    const crazysdk = window.CrazyGames.CrazySDK.getInstance();",
+            "    crazysdk.addEventListener(\"adblockDetectionExecuted\", adblockDetection);",
+            "    crazysdk.init();",
+            "} catch(e) {",
+            "    console.log({e});",
+            "}"
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Check if user is using an ad blocker.",
```

Comment:
Write that the action to scan for adblocks must be used before.

Reply:
updated